.nvmrc

```diff
@@ -1 +1 @@
-12
+16.14.0
```

Comment:
What's the benefit of having such a specific version. 

Would just `16` be sufficient?